sql/moz-fx-data-shared-prod/telemetry_derived/clients_daily_v6/metadata.yaml

```diff
@@ -1,21 +1,35 @@
----
-friendly_name: Clients Daily
-description: |-
-  A daily aggregation over `main` pings from each Firefox
+bigquery:
+  clustering:
+    fields:
+    - normalized_channel
+    - sample_id
+  time_partitioning:
+    field: submission_date
+    require_partition_filter: true
+    type: day
+description: 'A daily aggregation over `main` pings from each Firefox
```

Comment:
Can we restore the description to the `|-` version without the extra newlines?

Reply:
This was easier to configure than I thought.